docs/content/en/docs/getting-started/optional/api-server-extra-args.md

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
We need to update each of these pages to say that these additional settings can be configured

https://anywhere.eks.amazonaws.com/docs/getting-started/vsphere/vsphere-spec/
https://anywhere.eks.amazonaws.com/docs/getting-started/baremetal/bare-spec/
https://anywhere.eks.amazonaws.com/docs/getting-started/snow/snow-spec/
https://anywhere.eks.amazonaws.com/docs/getting-started/cloudstack/cloud-spec/
https://anywhere.eks.amazonaws.com/docs/getting-started/nutanix/nutanix-spec/